android/src/main/java/com/reactnativestripesdk/CardChangeEvent.kt

```diff
@@ -0,0 +1,56 @@
+package com.reactnativestripesdk
```

Comment:
This is a rename, not sure why git didn't pick it up. CardChangedEvent -> CardChangeEvent, for consistency.